shell/src/main/java/tachyon/shell/TfsShellUtils.java

```diff
@@ -94,7 +94,7 @@ public static String validatePath(String path, TachyonConf tachyonConf) throws I
    * @param tachyonClient the client used to fetch information of Tachyon files
    * @param inputURI the input URI (could contain wildcards)
    * @return a list of {@link TachyonURI}s that matches the inputURI
-   * @throws IOException
+   * @throws IOException if the client cannot open the directory or list its status
```

Comment:
It's a bit confusing, does `the directory` mean `inputURI`? Do you feel that `if any filesystem errors are encountered when expanding paths with wildcards` describes the intention?